errors.json

```diff
@@ -39,107 +39,96 @@
-		"http": 400,
-		"tags": [
-			"Processes"
+			"Data Processing"
 		]
 	},
 	"BudgetInvalid": {
 		"description": "The budget is too low as it is either smaller than or equal to 0 or below the costs.",
-		"message": "The budget is too low.",
+		"message": "The specified budget is too low.",
 		"http": 400,
 		"tags": [
-			"Job Management",
-			"Secondary Services Management"
+			"Batch Jobs",
+			"Secondary Services",
+			"Data Processing"
 		]
 	},
 	"NoDataForUpdate": {
 		"description": "For PATCH requests: No valid data specified at all.",
-		"message": "No valid data specified to be updated.",
+		"message": "No data specified to be updated.",
 		"http": 400,
 		"tags": [
-			"Job Management",
-			"Secondary Services Management",
-			"Process Graph Management"
+			"Batch Jobs",
+			"Secondary Services"
 		]
 	},
 	"PropertyNotEditable": {
 		"description": "For PATCH requests: The specified parameter can't be updated. It is read-only.",
-		"message": "Property '{property}' is read-only.",
+		"message": "The specified property '{property}' is read-only.",
 		"http": 400,
 		"tags": [
-			"Job Management",
-			"Secondary Services Management",
-			"Process Graph Management"
+			"Batch Jobs",
+			"Secondary Services"
 		]
 	},
 	"CollectionNotFound": {
 		"description": "The requested collection does not exist.",
 		"message": "Collection '{identifier}' does not exist.",
 		"http": 404,
 		"tags": [
-			"EO Data Discovery",
-			"Processes"
+			"EO Data Discovery"
 		]
 	},
 	"StorageFailure": {
 		"description": "Server couldn't store file(s) due to server-side reasons.",
-		"message": "Unable to store file(s).",
+		"message": "Unable to store files due to a server error. Please try again later.",
 		"http": 500,
 		"tags": [
 			"File Management",
-			"Job Management"
+			"Batch Jobs"
 		]
 	},
 	"StorageQuotaExceeded": {
 		"description": "The storage quota has been exceeded by the user.",
-		"message": "Insufficient Storage.",
+		"message": "Your storage quota has been exceeded.",
 		"http": 400,
 		"tags": [
 			"File Management",
-			"Job Management"
+			"Batch Jobs"
 		]
 	},
 	"FileNotFound": {
 		"description": "The requested file does not exist.",
 		"message": "File '{file}' does not exist.",
 		"http": 404,
 		"tags": [
-			"File Management",
-			"Processes"
+			"File Management"
 		]
 	},
 	"FilePathInvalid": {
 		"description": "The specified path is invalid or not accessible. Path could contain invalid characters, point to an existing folder or a location outside of the user folder.",
-		"message": "File path is invalid.",
+		"message": "File path is invalid: {reason}",
 		"http": 400,
 		"tags": [
 			"File Management"
 		]
 	},
 	"FileOperationUnsupported": {
 		"description": "The specified path is not a file and the operation is only supported for files. Path is likely a directory.",
-		"message": "Operation is only supported for files.",
+		"message": "Operation is only supported for files, not folders.",
```

Comment:
This was originally meant for this use case:

You upload `folder/file1.txt` and `folder/file2.txt` via `GET /files/folder/file{1,2}.txt` and then a client calls `DELETE /files/folder`, which the API does not allow, because we basically only work on files always. There are no other use cases that can occur from the file endpoints, are there?

Reply:
> There are no other use cases
probably from the point of of view of the abstract API, but a backend might want to use that error code for other  practical file system related issues as well.
Especially because the code "FileOperationUnsupported" sounds very generic

Reply:
I added a separate `FolderOperationUnsupported` error and changed `FileOperationUnsupported` to be a general-purpose error for file operations.